src/style.css

```
@@ -1 +1,70 @@
-/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

its forbidden to use anything except box-sizing  in *
```suggestion

}
```